daphne_worker_test/docker/miniflare.Dockerfile

```
@@ -1,27 +1,30 @@
 FROM rust:1.63-alpine AS builder
-WORKDIR /tmp/dap_test
-RUN apk add --update npm bash g++ openssl-dev
-RUN npm install -g n && \
+RUN apk add --update npm bash g++ openssl-dev git
+RUN npm install -g n wasm-pack && \
     n 18.4.0
-RUN npm install -g \
-        @cloudflare/wrangler \
-        wasm-pack \
-        miniflare@2.5.1
+
+WORKDIR /tmp/wrangler2
+RUN git clone https://github.com/cloudflare/wrangler2 -b wrangler@2.1.13 . && \
+    npm install -g wrangler
```

does this is enough here?
```suggestion
RUN npm install -g wrangler@2.1.13
```

---

Oh wow, that's much simpler. I missed this because the wrangler2 README only linked to the NPM page for v1.